frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -309,7 +309,7 @@ const ProjectListView: React.FC<IProps> = ({ filterable, title, mode }) => {
               </Button>
             </TooltipWrapper>
           )}
-          {keycloak.hasClaim(Claims.PROJECT_VIEW) && (
+          {keycloak.hasClaim(Claims.REPORTS_VIEW) && (
```

Comment:
Fixes bug where a user could click the Export button when they didn't have the appropriate role to actually do the export